deepface/modules/verification.py

```diff
@@ -239,7 +239,8 @@ def __extract_faces_and_embeddings(
     img_objs = detection.extract_faces(
         img_path=img_path,
         detector_backend=detector_backend,
-        grayscale=False,
+        color_face='rgb',
+        normalize_face=True,
```

Comment:
default values of color face and normalize face are rgb and True already, IMO we should not set them where it is being called if we are setting default values.

this is available also in representation, recognition,

Reply:
Removed them. I added it because grayscale was there.